nodewatcher/settings.py

```diff
@@ -250,6 +250,7 @@ def user_url(user):
 
     # Modules.
     #'nodewatcher.modules.analysis.rogue_nodes',
+    'nodewatcher.modules.analysis.channel_allocation',
```

Comment:
I think we should find a way to make test be skipped if its app it is not added to settings.